recipe/meta.yaml

```diff
@@ -36,21 +36,20 @@ requirements:
     - python
     - pip
     - pybind11
-    - pytorch
```

Comment:
this should not be removed

Reply:
So currently we have both pytorch with and without pin:
```
- pytorch
- pytorch =*={{ torch_proc_type }}*
```

Isn't it redundant to have the non-pinned version?

Reply:
no. the unpinned line is necessary for the bot to detect the package needs to be rebuilt for new pytorch versions.

Reply:
Oh interesting, thanks for explaining. Will revert (I'll squash everything after clarifying/fixing the `torch_proc_type` question below)